apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -2920,11 +2933,12 @@ suite('conversation.js >', function() {
           assert.isTrue(element.classList.contains('pending'));
         });
         test('click calls retrieveMMS', function() {
-          assert.isTrue(MessageManager.retrieveMMS.calledWith(message.id));
+          sinon.assert.calledWith(MessagingClient.retrieveMMS, message.id);
         });
         suite('response error', function() {
-          setup(function() {
-            MessageManager.retrieveMMS.returnValues[0].onerror();
+          setup(function(done) {
+            retrievePromise.catch(() => done());
```

Comment:
__optional nit:__ I'd place statement with ```done``` as the last statement in ```setup``` here and in tests below, but up to you.